hack/build.sh

```diff
@@ -16,6 +16,9 @@
 
 set -o pipefail
 
+[[ ! -v REPO_ROOT_DIR ]] && REPO_ROOT_DIR="$(git rev-parse --show-toplevel)"
```

Comment:
@dprotaso It's that -v which does not work in bash v3

No worries, I'll fix that tomorrow